js/data.js

```diff
@@ -9,40 +9,44 @@ function getPremierLeagueData() {
   xhr.responseType = 'json';
   xhr.addEventListener('load', function () {
     for (let i = 0; i < xhr.response.response[0].league.standings[0].length; i++) {
-      const $newTr = document.createElement('tr');
-      const $newTdBlank = document.createElement('td');
-      const $newTdRank = document.createElement('ins');
-      const $newTdLogo = document.createElement('img');
-      const $newTdTeam = document.createElement('ins');
-      const $newTdWin = document.createElement('td');
-      const $newTdDraw = document.createElement('td');
-      const $newTdLoss = document.createElement('td');
-      const $newTdPoints = document.createElement('td');
-      $teamStandings.appendChild($newTr);
-      $newTdBlank.setAttribute('class', 'rank-logo-name');
-      $newTr.appendChild($newTdBlank);
-      $newTdRank.textContent = xhr.response.response[0].league.standings[0][i].rank;
-      $newTdRank.setAttribute('class', 'rank');
-      $newTdBlank.appendChild($newTdRank);
-      $newTdLogo.setAttribute('src', xhr.response.response[0].league.standings[0][i].team.logo);
-      $newTdLogo.setAttribute('class', 'logo-img');
-      $newTdBlank.appendChild($newTdLogo);
-      $newTdTeam.textContent = xhr.response.response[0].league.standings[0][i].team.name;
-      $newTdBlank.appendChild($newTdTeam);
-      $newTdWin.textContent = xhr.response.response[0].league.standings[0][i].all.win;
-      $newTdWin.setAttribute('class', 'center-standing');
-      $newTr.appendChild($newTdWin);
-      $newTdDraw.textContent = xhr.response.response[0].league.standings[0][i].all.draw;
-      $newTdDraw.setAttribute('class', 'center-standing');
-      $newTr.appendChild($newTdDraw);
-      $newTdLoss.textContent = xhr.response.response[0].league.standings[0][i].all.lose;
-      $newTdLoss.setAttribute('class', 'center-standing');
-      $newTr.appendChild($newTdLoss);
-      $newTdPoints.textContent = xhr.response.response[0].league.standings[0][i].points;
-      $newTdPoints.setAttribute('class', 'center-standing');
-      $newTr.appendChild($newTdPoints);
+      standingsTable(xhr.response.response[0].league.standings[0][i]);
     }
   });
   xhr.send();
 }
 getPremierLeagueData();
+
+function standingsTable(xhr) {
+  const $newTr = document.createElement('tr');
+  const $newTdBlank = document.createElement('td');
+  const $newTdRank = document.createElement('ins');
+  const $newTdLogo = document.createElement('img');
+  const $newTdTeam = document.createElement('ins');
+  const $newTdWin = document.createElement('td');
+  const $newTdDraw = document.createElement('td');
+  const $newTdLoss = document.createElement('td');
+  const $newTdPoints = document.createElement('td');
+  $teamStandings.appendChild($newTr);
```

Comment:
This should happen outside of this function. So this should happen where you call `standingsTable()`. So you'll return the outtermost element inside this function and then wherever you call `standingsTable` that's where the appending happens